src/marqo/index.py

```diff
@@ -376,7 +376,7 @@ def recommend(self, documents: Union[List[str], Dict[str, float]],
 
```

Comment:
If we only accept `document` and `query` for content type, can you provide an Enum for it?

Reply:
EmbedContentType